Sources/KituraSession/SessionState.swift

```diff
@@ -76,6 +76,8 @@ public class SessionState {
             store.save(sessionId: id, data: data, callback: callback)
         } catch(let error as NSError) {
             callback(error)
+        } catch {
+            callback(nil)
```

Comment:
Under 3.0, if the `error` couldn't be cast to an `NSError`, the callback would never have be run at all (and the compiler didn't seem to care enough to complain), so at least this is better.

It's unarguably broken behavior, though. It's just not *_newly_* unarguably broken behavior.